src/Microsoft.Health.Fhir.Core/Features/Search/SearchOptions.cs

```diff
@@ -58,5 +58,10 @@ internal set
         /// Gets the list of search parameters that were not used in the search.
         /// </summary>
         public IReadOnlyList<Tuple<string, string>> UnsupportedSearchParams { get; internal set; }
+
+        /// <summary>
+        /// Gets the list of sorting parameters. The second item in a tuple is whether the sort is accending or decending.
+        /// </summary>
+        public IEnumerable<Tuple<string, string>> Sort { get; internal set; }
```

Comment:
We should define the sort order as an enum (I know it's redundant but string is not friendly here). Eventually, i think the FHIR .NET library will have a base library that contains all "common" objects that would work across versions so we can switch to that later.